### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by goedi02**
Question for reviewers: Should I move the information from the "conclusion" to the start of the file? To me this seems to be information that would be useful to know from the beginning. It could be integrated into the list of the three environments in the intro.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -94,11 +95,11 @@ This should trigger the same error as the one found in `docker logs` from the pr
 /usr/bin/nsjail --config /config/nsjail.cfg -Mo -v
 ```
 
-This should output more errors, that should provide you with more details about what went wrong.
+This should output more errors, and these errors should provide more details about what went wrong.
 
 ## Troubleshooting with Kubernetes
 
-If everything works in docker, the problem might be higher up (in Kubernetes). The first step to debug this would be to check if the challenge works in a local cluster. Follow the instructions [here](local-testing.md#running-the-challenge-in-kubernetes) for getting the task running in KIND.
+If everything works in Docker, the problem might be higher up (in Kubernetes). The first step to debug this would be to check if the challenge works in a local cluster. Follow the instructions [here](local-testing.md#running-the-challenge-in-kubernetes) for getting the task running in KIND.
```
Comment (sroettger) [inline docs/troubleshooting.md:102]: for KIND, maybe link to https://github.com/kubernetes-sigs/kind

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -60,18 +61,18 @@ If the challenge is a pwnable, you might see an error formatted like this:
 [E][2020-02-02T20:20:02+0200][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():204 execve('/bin/sh') failed: No such file or directory
 ```
 
-That's an NsJail error. In this case, it is complaining that `/bin/sh` failed and gave the error *"No such file or directory"*.
+That's an nsjail error. In this case, it is complaining that `/bin/sh` failed and returned the error *No such file or directory*.
```
Comment (sroettger) [inline docs/troubleshooting.md:64]: I also think nsjail looks better in lowercase (and the command is written this way) but just fyi https://github.com/google/nsjail uses the NsJail capitalization.
But still, +1 for lower case :)

**Comment by sroettger** [comment]
> Question for reviewers: Should I move the information from the "conclusion" to the start of the file? To me this seems to be information that would be useful to know from the beginning. It could be integrated into the list of the three environments in the intro.

Yeah, I think that would be useful in the beginning.

File: docs/troubleshooting.md (hunk 1 above)
Comment (goedi02) [reply docs/troubleshooting.md:102]: Done

File: docs/troubleshooting.md (hunk 2 above)
Comment (goedi02) [reply docs/troubleshooting.md:64]: Thanks for the info! I agree with sticking with lower case here; as long as the kCTF  doc is consistent, it's fine. When reviewing the Google Cloud walkthrough with Eduardo, we also opted for the lower case spelling.

**Comment by goedi02** [comment]
> > Question for reviewers: Should I move the information from the "conclusion" to the start of the file? To me this seems to be information that would be useful to know from the beginning. It could be integrated into the list of the three environments in the intro.
> 
> Yeah, I think that would be useful in the beginning.

Done and committed to the branch

**Pull request merged**